base/applications/notepad/main.c

```diff
@@ -602,25 +600,14 @@ int WINAPI _tWinMain(HINSTANCE hInstance, HINSTANCE prev, LPTSTR cmdline, int sh
         return 1;
     }
 
-    /* Setup windows */
-
-    monitor = MonitorFromRect(&Globals.main_rect, MONITOR_DEFAULTTOPRIMARY);
-    info.cbSize = sizeof(info);
-    GetMonitorInfoW(monitor, &info);
-
-    x = Globals.main_rect.left;
-    y = Globals.main_rect.top;
-    if (!IntersectRect(&rcIntersect, &Globals.main_rect, &info.rcWork))
-        x = y = CW_USEDEFAULT;
-
     /* Globals.hMainWnd will be set in WM_CREATE handling */
-    CreateWindow(className,
+    hwnd = CreateWindow(className,
```

Comment:
Isn't the `hwnd` basically what's being stored in `Globals.hMainWnd` ? If so, you may keep the code as it was, and use the globals hMainWnd below.